### PR TITLE
Document extra admin setting when running sandbox with authentication

### PR DESCRIPTION
**Pull request by pbatko-da**

### CODE REVIEW

File: docs/2.7.0/docs/tools/sandbox.rst [hunk 1]
```diff
@@ -103,6 +103,20 @@ with SHA-256) with the public key loaded from the given JWKS URL.
           TLS with server authentication as described in :ref:`sandbox-tls` for
           any request sent to the Ledger API in production.
 
+Fresh participant nodes come with the default participant admin user called ``participant_admin`` which
```
Comment (mziolekda) [inline docs/2.7.0/docs/tools/sandbox.rst:106]:
```suggestion
Fresh participant nodes come with a default participant admin user called ``participant_admin`` which
```

File: docs/2.7.0/docs/tools/sandbox.rst [hunk 2]
```diff
@@ -103,6 +103,20 @@ with SHA-256) with the public key loaded from the given JWKS URL.
...
+can be used to bootstrap other users.
+You might prefer to have an admin user with a different user id ready on the participant startup.
+For such situations you can specify an additional participant admin user with the user id of your choice.
```
Comment (mziolekda) [inline docs/2.7.0/docs/tools/sandbox.rst:109]:
```suggestion
Fresh participant nodes come with a default participant admin user called ``participant_admin``, which
can be used to bootstrap other users.
You might prefer to have an admin user with a different user id ready on a participant startup.
For such situations you can specify an additional participant admin user with the user id of your choice.
```

**Review by mziolekda — APPROVED** [review]
Small corrections, but otherwise looks good. It would be good to have someone from the doc team to review

File: docs/2.7.0/docs/tools/sandbox.rst [hunk 3]
```diff
@@ -103,6 +103,20 @@ with SHA-256) with the public key loaded from the given JWKS URL.
           TLS with server authentication as described in :ref:`sandbox-tls` for
           any request sent to the Ledger API in production.
 
+Fresh participant nodes come with a default participant admin user called ``participant_admin``, which
+can be used to bootstrap other users.
+You might prefer to have an admin user with a different user id ready on a participant startup.
+For such situations you can specify an additional participant admin user with the user id of your choice.
```
Comment (loiskilberg-da) [inline docs/2.7.0/docs/tools/sandbox.rst:109]:
```suggestion
For such situations, you can specify an additional participant admin user with the user id of your choice.
```

File: docs/2.7.0/docs/tools/sandbox.rst [hunk 4]
```diff
@@ -103,6 +103,20 @@ with SHA-256) with the public key loaded from the given JWKS URL.
...
+
+.. note:: If a user with the specified id already exists then no additional user will be created,
```
Comment (loiskilberg-da) [inline docs/2.7.0/docs/tools/sandbox.rst:111]:
```suggestion
.. note:: If a user with the specified id already exists, then no additional user will be created,
```

File: docs/2.7.0/docs/tools/sandbox.rst [hunk 5]
```diff
@@ -103,6 +103,20 @@ with SHA-256) with the public key loaded from the given JWKS URL.
...
+          even if the pre-existing user was not an admin user.
```
Comment (loiskilberg-da) [inline docs/2.7.0/docs/tools/sandbox.rst:112]:
```suggestion
          even if the preexisting user was not an admin user.
```

**Review by loiskilberg-da — APPROVED** [review]

**Pull request merged**